test/CMakeLists.txt

```diff
@@ -6,6 +6,8 @@
 # http://www.boost.org/LICENSE_1_0.txt
 #---------------------------------------------------------------------------#
 
+include(CMTest)
+
```

Comment:
Redundant, already included in upper CMakeLists

Reply:
I use solana repo as an umbrella repo for local development. It doesn't compiler there without this. Probably if I add it in the upper level CMakeLists file, it will work, but let's keep it here because it's easier this way.